05_syntax_lab/01.py

```diff
@@ -2,4 +2,14 @@
 Write a program that reads 10 numbers from
 the user and prints the largest one
 """
+# Exercise 1
+# By Zvi Figov
+print "Enter 10 positive numbers" # Assuming positive numbers 
+largest = -1
+for i in range(1,11):
+    print "Enter number %d" % i
+    num = int(raw_input())
+    largest = max(num,largest)
+
+print "Largest number is %d" % largest
```

Comment:
Welcome to the exercises!

Your solution works in some of the cases, but what if all the numbers are less than -10? Could you fix the code so there won't be such limitation?

In addition, when you prompt the user to enter a number, the cursor is on the line following the prompt. Could you change that so the prompt will be on the same line as the "Enter number 2" text ?